docs/src/network.md

```diff
@@ -421,6 +421,17 @@ await page.RouteAsync("**/api/fetch_data", async route => {
 await page.GotoAsync("https://example.com");
 ```
 
+#### Precedence
+
+When a request matches multiple routes, the last matched route is used to handle the request.
+
+```js
+page.route("**/api", route => route.abort());
+page.route("**/api/useful-resource", route => route.fulfill(/* data */));
+```
+
+The above routes will return data for `useful-resource`, and block all other requests to the API.
```

Comment:
Do you want to also mention the `route.fallback` here?

Reply:
Great idea. Added, and reworded the text a little. 

Not sure the link is formatted correctly, I looked at how other links are defined and tried to follow the same convention, let me know if it needs changing.